epdb/epdb_client.py

```diff
@@ -143,13 +146,20 @@ def interact(self):
                         print('*** Connection closed by remote host ***')
                         break
                     if text:
-                        sys.stdout.write(text.decode('ascii'))
+                        all_text = remaining_text + text
+                        try:
+                            unicode_text = all_text.decode(encoding)
+                            remaining_text = b""
+                        except UnicodeDecodeError as e:
+                            unicode_text = all_text[:e.start].decode(encoding)
+                            remaining_text = all_text[e.start:]
```

Comment:
I just had to use this patch again to realize that we never finished this PR.

For e.start < 0 - I accepted my own assertion above since I think it should simply never happen.

For e.start == 0 -- I think it is unlikely but legit case whenever we read only few bytes which are beginning of an incomplete unicode.  Then we just store it all and append to next block -- I do not see how we could get an infinite loop here since we are not adding any looping and should exit at eof (would still do) ... the only thing would be left is at the end to assert that we have no remaining_text left -- as if incomplete unicode at the end was provided and we failed to decode it.  I will add a stab for that.